### PR TITLE
v4.3.0: add default.xml and release notes

### PR DESCRIPTION
**Pull request by KanjiMonster**
*** THIS PR MUST NOT BE REBASED ***

Set the relese FEEDURIPREFIX, add a default.xml based on the nightlies and add release-notes for this release.

Since the default.xml directly references the preceeding commit, this PR must not be rebased on merge, else the default.xml will break.

Rebasing/fixing any commits after the first one is fine.

Signed-off-by: Jonas Gorski <jonas.gorski@bisdn.de>

### CODE REVIEW

**Comment by KanjiMonster** [comment]
Fixed the meta-switch commit hash (picked the wrong target to use the default.xml), fixed the version in the release notes, and added the "Updated packages" header again.

File: release-notes.txt [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+4.3.0 Release Notes
+Date: 2021-11-09
+
+! New in this release
+- BISDN Linux now ships with a default en-us locale, allowing applications
+  that rely on locales like tmux to work.
+- Changing a bridge VLAN's egress tagged or pvid state during runtime is now
+  properly handled.
+- Accton AS4610: Added support for the AS4610-54T (B) model with reversed
+  airflow.
```
Comment (jklare) [inline release-notes.txt:11]:
```suggestion
  airflow. Special thanks to Ralf Kundel for helping with testing the compatibility.
```

**Review by jklare — COMMENTED** [review]
lgtm, one suggestion

**Review by jklare — APPROVED** [review]
lgtm

**Pull request merged**